web-api/src/business/useCases/workItems/assignWorkItemsInteractor.ts

```diff
@@ -71,6 +70,23 @@ export const assignWorkItemsInteractor = async (
     workItemEntity = new WorkItem(workItem);
   }
 
+  const docketEntry = (
```

Comment:
The only reason we call this is to be able to pass in the document title. How necessary is this? Can we know that the user being assigned to is always petitions or docket? If so, we can get rid of this.

Reply:
I suppose, in principle, we cannot assume that. :/